tests/test_coco_iris_to_kvp_wrapper/test_detection_as_kvp.py

```diff
@@ -2,7 +2,10 @@
 
 from tests.test_fixtures import DetectionTestFixtures
 from vision_datasets.common.constants import DatasetTypes
-from vision_datasets.image_object_detection import DetectionAsKeyValuePairDataset, DetectionAsKeyValuePairDatasetForMultilabelClassification
+from vision_datasets.image_object_detection import (
+    DetectionAsKeyValuePairDataset,
+    DetectionAsKeyValuePairDatasetForMultilabelClassification,
```

Comment:
Nit: maybe no comma at the end of the last element?

Reply:
Hm, looks like `ruff` formatter automatically adds a trailing comma.

Reply:
Ah, then nevermind :D!